railties/test/application/action_text/installation_integration_test.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require "isolation/abstract_unit"
+require "rack/test"
+
+module ApplicationTests
+  class InstallationIntegrationTest < ActiveSupport::TestCase
+    include ActiveSupport::Testing::Isolation
+    include Rack::Test::Methods
+
+    setup :build_app
+    teardown :teardown_app
+
+    test "renders actionable exception page when Action Text is not installed" do
+      add_to_config <<~RUBY
+        config.active_record.migration_error    = :page_load
+        config.consider_all_requests_local      = true
+        config.action_dispatch.show_exceptions  = :all
+      RUBY
+
+      app_file "config/routes.rb", <<~RUBY
+        Rails.application.routes.draw do
+          post "/blobs" => "blobs#create"
+          post "/rich_texts" => "rich_texts#create"
+        end
+      RUBY
+
+      app_file "app/controllers/blobs_controller.rb", <<~RUBY
+        class BlobsController < ActionController::Base
+          def create
+            ActiveStorage::Blob.create!
+          end
+        end
+      RUBY
+
+      app_file "app/controllers/rich_texts_controller.rb", <<~RUBY
+        class RichTextsController < ActionController::Base
+          def create
+            ActionText::RichText.create!
+          end
+        end
+      RUBY
+
+      app "development"
+
+      post "/rich_texts"
+
+      assert_equal 500, last_response.status
+      assert_match "To resolve this issue run: bin/rails action_text:install", last_response.body
+
+      post "/blobs"
+
+      assert_equal 500, last_response.status
```

Comment:
Is it possible last_response could get corrupted between requests?
Should these be separate test cases with the setup appending config, routes, controllers, etc?